Assets/Scripts/DataController.cs

```diff
@@ -277,7 +269,19 @@ public void UpdateOnSceneLoad()
                 starComponent.GetComponent<Renderer>().material = starMaterial;
             }
         }
-        if (showHorizonView) positionNCP();
+        
+        // if we're in Horizon view, set location and date / time
+        if (showHorizonView)
+        {
+            if (manager.UserHasSetLocation)
+            {
+                nextLocation = manager.LocalUserPin.Location;
+                // Force an update once ready
+                currentLocation = new LatLng();
```

Comment:
Seems like we need to  set time here eventually.

Reply:
This _should_ be picked up from the manager's CurrentSimulationTime, though there are issues there to be resolved...